multiqc/modules/rseqc/bam_stat.py

```diff
@@ -35,7 +35,8 @@ def parse_reports(self):
         'reads_mapped_in_proper_pairs': r"Reads mapped in proper pairs:\s*(\d+)",
         'proper-paired_reads_map_to_different_chrom': r"Proper-paired reads map to different chrom:\s*(\d+)",
     }
-    
+    #intiate PE check
+    PE=False
```

Comment:
I'd prefer to use slightly more descriptive variable names generally (eg. `is_paired_end`)